Mathlib/Probability/Kernel/MeasureCompProd.lean

```diff
@@ -58,6 +58,11 @@ lemma compProd_apply [SFinite μ] [IsSFiniteKernel κ] {s : Set (α × β)} (hs
   simp_rw [compProd, Kernel.compProd_apply hs, Kernel.const_apply, Kernel.prodMkLeft_apply']
   rfl
 
+@[simp]
+lemma compProd_apply_univ [SFinite μ] [IsMarkovKernel κ] : (μ ⊗ₘ κ) .univ = μ .univ := by
+  rw [compProd_apply .univ]
```

Comment:
```suggestion
lemma compProd_apply_univ [SFinite μ] [IsMarkovKernel κ] : (μ ⊗ₘ κ) univ = μ univ := by
  rw [compProd_apply MeasurableSet..univ]
```
What about opening `Set` at the top of the file to get rid of the weird `.univ`?